### PR TITLE
Update WOTLK_BC_Terokkar_Forest.lua

### PR DESCRIPTION
**Pull request by Hendo72**
guide flow tweaks

### CODE REVIEW

File: WoWPro_Leveling/Wrath/Horde/WOTLK_BC_Terokkar_Forest.lua [hunk 1]
```diff
@@ -60,12 +60,12 @@ A Wanted: Bonelashers Dead!|QID|10034|M|49.82,45.27|N|From the WANTED Poster.|
 T News for Rakoria|QID|10105|M|50.13,44.88|N|To Rakoria.|
 A Olemba Seed Oil|QID|9993|M|50.13,44.88|N|From Rakoria.|
 A Vestments of the Wolf Spirit|QID|10018|M|50.17,46.44|N|From Malukaz.|
-C Vestments of the Wolf Spirit|QID|10018|M|56.5,41.4|N|[Coords]Kill Timber Worg (Alpha)s|S|
 C Olemba Seed Oil|QID|9993|L|25745 30|N|[color=FF0000]NOTE: [/color]The Olemba Cones are found on the ground spread all across Terokkar.|S|
-K Magical Disturbances|ACTIVE|10027|M|56.5,41.4|QO|1|
-C Vestments of the Wolf Spirit|QID|10018|M|56.5,41.4|N|[Coords]Kill Timber Worg (Alpha)s|US|
-R Veil Shienor|QID|10847|M|59.73,25.51|N|[Verify Coords]Head northeast to Veil Shienor.|
-C Arakkoa War Path|QID|10868|N|Kill Arakkoa until you complete the quest.|S|
+K Magical Disturbances|ACTIVE|10027|M|57.54,39.69|QO|1|N|[color=FF0000]NOTE: [/color]They can become invisible.|S|
```
Comment (Ludovicus-Maior) [inline WoWPro_Leveling/Wrath/Horde/WOTLK_BC_Terokkar_Forest.lua:64]: `S!US` ?

File: WoWPro_Leveling/Wrath/Horde/WOTLK_BC_Terokkar_Forest.lua [hunk 2]
```diff
@@ -60,12 +60,12 @@ A Wanted: Bonelashers Dead!|QID|10034|M|49.82,45.27|N|From the WANTED Poster.|
...
+C Vestments of the Wolf Spirit|QID|10018|M|56.5,41.4|L|25812 12|N|Any Timber Worg\n[color=FF0000]NOTE: [/color]They spread out over the area and depending on how busy it is, you may have to look for them.|
+K Magical Disturbances|ACTIVE|10027|M|57.54,39.69|QO|1|N|[color=FF0000]NOTE: [/color]They can become invisible.|US|
+R Veil Shienor|QID|10847|M|59.73,25.51|N|mHead northeast to Veil Shienor.|
+K Arakkoa War Path|ACTIVE|10868|QO|1;2|N|They are located at either Veil Shienor or.|S|
```
Comment (Ludovicus-Maior) [inline WoWPro_Leveling/Wrath/Horde/WOTLK_BC_Terokkar_Forest.lua:68]: `or` what?

File: WoWPro_Leveling/Wrath/Horde/WOTLK_BC_Terokkar_Forest.lua [hunk 3]
```diff
@@ -60,12 +60,12 @@ A Wanted: Bonelashers Dead!|QID|10034|M|49.82,45.27|N|From the WANTED Poster.|
...
-C Vestments of the Wolf Spirit|QID|10018|M|56.5,41.4|N|[Coords]Kill Timber Worg (Alpha)s|S|
 C Olemba Seed Oil|QID|9993|L|25745 30|N|[color=FF0000]NOTE: [/color]The Olemba Cones are found on the ground spread all across Terokkar.|S|
-K Magical Disturbances|ACTIVE|10027|M|56.5,41.4|QO|1|
-C Vestments of the Wolf Spirit|QID|10018|M|56.5,41.4|N|[Coords]Kill Timber Worg (Alpha)s|US|
-R Veil Shienor|QID|10847|M|59.73,25.51|N|[Verify Coords]Head northeast to Veil Shienor.|
-C Arakkoa War Path|QID|10868|N|Kill Arakkoa until you complete the quest.|S|
+K Magical Disturbances|ACTIVE|10027|M|57.54,39.69|QO|1|N|[color=FF0000]NOTE: [/color]They can become invisible.|S|
+C Vestments of the Wolf Spirit|QID|10018|M|56.5,41.4|L|25812 12|N|Any Timber Worg\n[color=FF0000]NOTE: [/color]They spread out over the area and depending on how busy it is, you may have to look for them.|
+K Magical Disturbances|ACTIVE|10027|M|57.54,39.69|QO|1|N|[color=FF0000]NOTE: [/color]They can become invisible.|US|
+R Veil Shienor|QID|10847|M|59.73,25.51|N|mHead northeast to Veil Shienor.|
```
[the remark this answers is not in the document]
Comment (Hendo72) [reply WoWPro_Leveling/Wrath/Horde/WOTLK_BC_Terokkar_Forest.lua:67]: Wrong screen when i tried to open the map. I do it all the time :blush:

File: WoWPro_Leveling/Wrath/Horde/WOTLK_BC_Terokkar_Forest.lua (hunk 2 above)
Comment (Hendo72) [reply WoWPro_Leveling/Wrath/Horde/WOTLK_BC_Terokkar_Forest.lua:68]: Incomplete thought. I'm headed to 'or' now.

File: WoWPro_Leveling/Wrath/Horde/WOTLK_BC_Terokkar_Forest.lua (hunk 1 above)
Comment (Hendo72) [reply WoWPro_Leveling/Wrath/Horde/WOTLK_BC_Terokkar_Forest.lua:64]: for which?

Comment (Ludovicus-Maior) [reply WoWPro_Leveling/Wrath/Horde/WOTLK_BC_Terokkar_Forest.lua:64]: It is used on the `|S|` step and the `|US|` step goes away.

**Pull request merged**